apps/music/views/artist-detail/view.js

```diff
@@ -17,33 +17,28 @@ var ArtistDetailView = View.extend(function ArtistDetailView() {
   });
 
   this.list.configure({
-    getSectionName: (item) => {
-      return item.section;
-    }
+    getSectionName: (item) => item.section
```

Comment:
nit: When doing a one-param, one-line fat-arrow function, leave out the parenthesis -- `item => item.section`